ext/winevt/winevt_utils.cpp

```diff
@@ -88,59 +90,60 @@ guid_to_wstr(const GUID& guid)
 VALUE
 get_values(EVT_HANDLE handle)
 {
-  std::vector<WCHAR> buffer;
+  VALUE vbuffer;
+  EVT_VARIANT *buffer;
```

Comment:
nits: The latter variable uses `PEVT_VARIANT` as `EVT_VARIANT *` type.
```
PEVT_VARIANT pRenderedValues = buffer;
```

Shouldn't we unify `PEVT_VARIANT` usage?
This glitch is not important to handle memory properly.
Because the following equation is correct:
`EVT_VARIANT *` = `PEVT_VARIANT`